environment.yml

```diff
@@ -0,0 +1,14 @@
+name: pubtator
+dependencies:
+- beautifulsoup4=4.5.1
```

Comment:
you can get rid of the packages we're not using... which is most of these. Also add python and the version you want.